core/src/main/java/org/apache/iceberg/deletes/Deletes.java

```diff
@@ -125,12 +100,6 @@ public static PositionDeleteIndex toPositionBitmap(CloseableIterable<Long> posDe
     }
```

Comment:
Should this method be named `toPositionDeleteIndex`? The way `PositionDeleteIndex` is modeled, it could be implemented by a set.